src/main/java/seedu/address/logic/commands/SpecifyPathCommand.java

```diff
@@ -0,0 +1,52 @@
+//@@author A0124591H
+
+package seedu.address.logic.commands;
+
+import java.io.IOException;
+
+import seedu.address.commons.core.Config;
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.commons.util.ConfigUtil;
+import seedu.address.storage.XmlToDoAppStorage;
+
+/**
+ * Changes the file path of the ToDoApp
+ */
+public class SpecifyPathCommand extends Command {
+
+    public static final String COMMAND_WORD = "cd";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Changes the file path for the ToDoApp."
+            + "Parameters: FILE_PATH\n"
+            + "Example: " + COMMAND_WORD + " ~/ToDoApp/ToDoApp.xml";
+
+    public static final String MESSAGE_SUCCESS = "File path successfully changed to: %1$s";
+    public static final String MESSAGE_FAIL = "Error changing file path";
+    public static final String MESSAGE_IO = "Error with i/o";
+    public static final String MESSAGE_DATA_CONVERSION = "Error changing configuration";
+
+    private final String filePath;
+
+    public SpecifyPathCommand(String filePath) {
+        this.filePath = filePath;
+    }
+
+    @Override
+    public CommandResult execute(){
```

Comment:
Can consider writing some inline comments for these - else it'll be quite hard to understand what is going on under the hood